addon/v1alpha1/types_clustermanagementaddon.go

```diff
@@ -0,0 +1,65 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:scope="Cluster"
+// +kubebuilder:printcolumn:name="DISPLAY NAME",type=string,JSONPath=`.spec.displayName`
+// +kubebuilder:printcolumn:name="CRD NAME",type=string,JSONPath=`.spec.addOnConfigCRD`
+
+// ClusterManagementAddOn represents the registration of an add-on to the cluster management.
```

Comment:
What operator CRD does this API have behavior with respect to?

Reply:
can u clarify this comment